kanidmd/src/lib/be/dbvalue.rs

```diff
@@ -80,6 +80,11 @@ pub struct DbValueTaggedStringV1 {
     pub d: String,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DbValueEmailAddressV1 {
+    pub d: String,
```

Comment:
The intent is later to extra metadata. Remember, this type of dbvalue is "how it will be encoded into the db" so we minimise the struct field names because they are serialised and this reduces our IO in/out. So we have the "email" in its raw-serialised form, and the higher level Value type is out "rich" type that *could* have an EmailAddr type, if one existed. 

Saying this, the Value type needs a rewrite - like all code, it has some techdebt and poor choice :)